config/locales/en.yml

```diff
@@ -22,3 +22,6 @@
 en:
   hello: "Hello world"
   sensitive_message: "Culturally sensitive content: This is an image of a person or persons now deceased. In some Aboriginal Communities, hearing names or seeing images of deceased persons may cause sadness or distress, particularly to the relatives of these people.  Would you like to view its content?"
+  date_labels: {
```

Comment:
Hi Matt,
It's done. See https://github.com/ucsdlib/damspas/commit/1267800c2edf2879ddea2f8055ff6ca0b7d13067.